### DISCUSSION TITLE
Configuring Web server

### DISCUSSION THREAD

**Discussion by david-steg** [nextcloud/all-in-one#608] [Ideas] [answered]
Add Documentation for any changes in php. 
Without break anythings. 
For exemple change "upload_max_filesize 10G"

Using
`/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/.user.ini`?
OR
`/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/.htaccess`?

**Comment by szaimen** [comment, marked as answer]
Hello, 

We will not add a way to modify this because the project needs to be maintainable and certain things need to be fixed for that.

However you might be able to modify the php.ini file persistantly by mounting your modified php.ini into the Nextcloud container by using https://github.com/nextcloud/all-in-one/tree/main/manual-install.
